helpers_test.go

```diff
@@ -0,0 +1,729 @@
+package fsevents
+
+import (
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"runtime"
+	"sort"
+	"strconv"
+	"strings"
+	"sync"
+	"testing"
+	"time"
+)
+
+// EventFlags extensions for tests.
+
+var eventFlagsPossible = map[string]EventFlags{
+	"MustScanSubDirs":   MustScanSubDirs,
+	"KernelDropped":     KernelDropped,
+	"UserDropped":       UserDropped,
+	"EventIDsWrapped":   EventIDsWrapped,
+	"HistoryDone":       HistoryDone,
+	"RootChanged":       RootChanged,
+	"Mount":             Mount,
+	"Unmount":           Unmount,
+	"ItemCreated":       ItemCreated,
+	"ItemRemoved":       ItemRemoved,
+	"ItemInodeMetaMod":  ItemInodeMetaMod,
+	"ItemRenamed":       ItemRenamed,
+	"ItemModified":      ItemModified,
+	"ItemFinderInfoMod": ItemFinderInfoMod,
+	"ItemChangeOwner":   ItemChangeOwner,
+	"ItemXattrMod":      ItemXattrMod,
+	"ItemIsFile":        ItemIsFile,
+	"ItemIsDir":         ItemIsDir,
+	"ItemIsSymlink":     ItemIsSymlink,
```

Comment:
Can these just be "created", "removed", "isdir", etc. without the "item" every time? It's very repetitive and "noisy" in the test files.

Reply:
Hmm, I can change it, it is no problem, but I made it the same as the name of the const to immediately know what is being referred to here. Also makes it grep-able without the extra step. 
It is a bit of a crappy solution admittedly.

Don't have a strong opinion though. If you'd like, I can change it into more human readable names.

Reply:
It's not the biggest thing in the world; you can leave it if that's what you prefer.